specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx

```diff
@@ -157,6 +167,16 @@ export function TreeViewActions<SCHEMA extends AnyTree>({
           />
         </li>
       )}
+      {hasPermission(resourceName, 'bulkMove') && tableName === 'Storage' ? (
```

Comment:
(optional) 

Can you swap the boolean expressions here? 
It is far more likely that the `tableName === 'Storage'` check is faster than checking permissions, so we can take advantage of [short-circuit evaluation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation). 

```suggestion
      {tableName === 'Storage' && hasPermission(resourceName, 'bulkMove') ? (
```